### ISSUE TITLE
If there is a failure scenario, i want to continue test suite

### ISSUE THREAD

**Issue by Emre-ee** [ThrowTheSwitch/Ceedling#793]
Hi,

When I test my project, test suite does not continue if there is a failure scenario.  I used Google Test framework and there is called EXPECT_EQ(expect, actual) macro. You can use it for this purpose. Do you think  add new features to Unity like TEST_EXPECT_EQUAL(expect,actual) ?

**Comment by mvandervoord** [comment]
Yes. I believe there is an open issue for this already? If not, it's definitely something we've talked about doing. :)

**Comment by hiSandog** [comment]
A non-fatal expectation needs more than a macro alias: Unity must record the failure, return control to the test body, and still run teardown while preserving the first/all failure locations. It should remain distinct from existing fatal assertions so users can choose whether continuing is safe after a mismatch. Tests should cover several expectations in one test, an expectation followed by a crash-prone operation, and suite exit status/report counts.
